cmd/utils/manifest.go

```diff
@@ -268,17 +276,103 @@ func (*Executor) Execute(command string, env []string) error {
 	done := make(chan struct{})
 	scanner := bufio.NewScanner(r)
 
-	go func() {
-		for scanner.Scan() {
-			line := scanner.Text()
-			fmt.Println(line)
-		}
-		done <- struct{}{}
-	}()
+	if e.outputMode == "plain" {
+		go displayPlainOutput(scanner, done)
+	} else {
+		go commandOutput(scanner, done, command)
+	}
 
 	if err := cmd.Start(); err != nil {
 		return err
 	}
 
 	return cmd.Wait()
 }
+
+func displayPlainOutput(scanner *bufio.Scanner, done chan struct{}) {
```

Comment:
as the executor logic is growing with these changes, would it make sense to create a `executor.go` file in the same package moving all the executor logic there?

I didn't do it when I created this file because the logic was too small, but probably it will grow more